readme.md.md

```diff
@@ -0,0 +1,2 @@
+# CodeForCause 
+#gtrw
```

Comment:
Sorry I did't know Hacktober Fest got started and until then i was taking tutorial from CodeForCause youtube and just making my pull request and practicing it, and it got counted toward one of my PR in hacktoberfest.So please dont report it ,it just was an accident and still im a newbie and learning it .
Thanks